pkg/base/store_spec.go

```diff
@@ -369,12 +369,23 @@ func PreventedStartupFile(dir string) string {
 	return filepath.Join(dir, "_CRITICAL_ALERT.txt")
 }
 
-// GetPreventedStartupMessage attempts to read the PreventedStartupFile for each
-// store directory and returns their concatenated contents. These files
-// typically request operator intervention after a corruption event by
-// preventing the affected node(s) from starting back up.
-func (ssl StoreSpecList) GetPreventedStartupMessage() (string, error) {
-	var buf strings.Builder
+// AssertNoPriorCriticalAlert attempts to read the
+// PreventedStartupFile for each store directory and returns their
+// contents as a structured error.
+//
+// These files typically request operator intervention after a
+// corruption event by preventing the affected node(s) from starting
+// back up.
+func (ssl StoreSpecList) AssertNoPriorCriticalAlert() (err error) {
```

Comment:
The `Assert` here strikes me as weird, since the "assertion" is returned via an error. Have you considered something like `PriorCriticalAlertError()`?

Reply:
I'll take it. Thanks for the suggestion.